### PR TITLE
Add a page for editing scrambles

### PR DESCRIPTION
**Pull request by gregorbg**
Shameless copy/paste from the 'Edit Results' component, but we have to make do for the time being.
We should probably clean up and unify both components in one go anyways.

# Screenshots
image
image
image

### CODE REVIEW

File: app/controllers/admin/scrambles_controller.rb [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+module Admin
+  class ScramblesController < AdminController
+    # NOTE: authentication is performed by admin controller
+
+    def new
+      competition = Competition.find(params[:competition_id])
+      round = Round.find(params[:round_id])
+      # Create some basic attributes for that empty scramble.
+      # Using Scramble.new wouldn't work here: we have no idea what the country
```
Comment (maxidragon) [inline app/controllers/admin/scrambles_controller.rb:11]: What do you mean by "country" here?

Comment (gregorbg) [reply app/controllers/admin/scrambles_controller.rb]: Whoops, you caught me in the act of copy-pasting :see_no_evil:

**Comment by gregorbg** [comment]
@thewca-bot deploy staging

**Comment by gregorbg** [comment]
@thewca-bot deploy staging

**Comment by gregorbg** [comment]
@thewca-bot deploy staging

**Comment by danieljames-dj** [comment]
@thewca-bot deploy staging

**Comment by danieljames-dj** [comment]
@thewca-bot deploy staging

**Comment by gregorbg** [comment]
Merging on my authority/responsibility.
Has been tested on staging by at least 2 WRT members.

**Pull request merged**